src/js/actions/store.js

```diff
@@ -0,0 +1,20 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from '../reducers/';
+import { createLogger } from 'redux-logger';
+import { persistStore, persistReducer } from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+
+const persistConfig = {
```

Comment:
Here you're persisting whole app state which is good ;) but when i have persisted state and I'll refresh a window i still have a movies that i was looking for BUT i can't see that search filter is applied and i have no indicator what phrase is used to search - i can see default "What are you looking for?"

So when state is persisted pleas keep proper search value in this textbox then i have an indicator that it's already filtered and one more thing - how to reset? right now i need to hit space and then search in textbox